resources/src/models/ManuscriptRecord/views/ManuscriptRecordFormView.vue

```diff
@@ -83,7 +83,7 @@ const generalSectionStatus = computed(() => {
     return 'error'
 
   if (manuscript.type === 'secondary') {
-    if (manuscript.no_ogl_explanation === '' && manuscript.do_not_apply_ogl)
+    if (manuscript.no_ogl_explanation === '' && manuscript.apply_ogl)
```

Comment:
This should be reverted as it will show incomplete when valid.
```suggestion
    if (manuscript.no_ogl_explanation === '' && !manuscript.apply_ogl)
```